plugins/main/public/controllers/management/components/management/configuration/cluster/cluster.js

```diff
@@ -58,10 +85,22 @@ class WzCluster extends Component {
     let mainSettingsConfig = {
       ...currentConfig['com-cluster'],
       disabled:
-        currentConfig['com-cluster'].disabled === 'yes'
-          ? 'disabled'
-          : 'enabled',
+        currentConfig['com-cluster'].disabled === true ? 'disabled' : 'enabled',
```

Comment:
**question**: did it change the value of the `disabled` property to a boolean?

Reply:
It seems to come from previous versions returning a boolean

<img width="1489" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/fc3383c2-54fe-414f-939d-5a72e79e8975">